libs/content/workspace/Cargo.toml

```diff
@@ -22,6 +22,7 @@ rand = "0.8.5"
 rayon = "1.10.0"
 resvg = "0.41.0"
 serde = { version = "1.0.171", features = ["derive"] }
+spellbook = { path = "../../../../../spellbook" }
```

Comment:
note: spelling suggestions only available on their master branch (sort of - I had to comment out a `todo!()` to stop it from crashing)